### ISSUE TITLE
[ACTIVE_STALE_TAKEOVER] SWARM-1565 — PR1514 measurement-root repair

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1565]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1565
STATUS: ACTIVE_STALE_TAKEOVER_PENDING_POSTCLAIM_DIRECT_ARBITRATION
SWARM_LANE_KEY: C01|PR1514|MODEL341-RESOURCE-ENVELOPE-MEASUREMENT-ROOT|V4
PREFERRED_LANE: #7 D06
PREFERRED_WORK_KIND: PRODUCT_VERTICAL
SELECTED_WORK_KIND: SAME_LINEAGE_STALE_TAKEOVER / PRODUCT_VERTICAL / REPRODUCIBILITY REPAIR
PARENT_ISSUES: #7 #13 #548 #723 #802 #1262 #1304 #1509 #1514 #1518 #1523 #1551
BASE_SHA: 9585ed286fb5a05cdd30cf4fd76fb08330934bef
TARGET_PR: #1514
TARGET_BRANCH: swarm/1509-model341-resource-envelope-v4
PREDECESSOR_PRODUCT_OWNER: #1509
PREDECESSOR_LAST_ISSUE_UPDATE_UTC: 2026-09-12T06:58:34Z
PREDECESSOR_PR_LAST_UPDATE_UTC: 2026-09-12T06:58:22Z
CURRENT_INTEGRATION_OWNER: #1551 or lawful successor
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS_AUTHORITY
LEASE_UNTIL_UTC: CLAIM_UPDATED_AT_PLUS_6_HOURS
LOCAL_FREE_ONLY: true

ROUTING:
Issue #1565 => preferred lane 2 + (1565 mod 15) = #7; preferred work kind WORK_KINDS[(1565 div 15) mod 8] = PRODUCT_VERTICAL. Lane #7 is evaluation/stage-gate work; this package is a launch-stage measured-resource evidence repair and fits Product/reproducibility work without taking D06 final-test or evaluation payload authority.

STALE-TAKEOVER BASIS:
1. #1509 last issue update is 2026-09-12T06:58:34Z and PR #1514 last update is 06:58:22Z. Its default six-hour live Product lease therefore expired before this claim.
2. PR #1514 remains OPEN/non-draft on exact unchanged head 9585ed286fb5a05cdd30cf4fd76fb08330934bef.
3. Exact-head shared CI is terminal SUCCESS, but independent SWARM-1518 verdict is CHANGES_REQUIRED, so the candidate must be repaired in the same lineage rather than audited or integrated.
4. #1523 discovered that exact prior verdict and explicitly abandoned/released duplicate audit ownership; it does not own Product repair.
5. Preclaim semantic search found no newer Product repair owner for PR #1514 after #1509. Mandatory postclaim direct OPEN issue+PR arbitration still decides ownership before Product mutation.
6. Training authority remains zero, so this closes one exact #548 pre-step resource-envelope integrity blocker without fabricating learned-target optimization.

EXACT FINDING TO CONSUME:
The v4 historical measurement tuple is not independently immutable when raw_samples_retained=false. validate_report() can accept a coherently self-resealed report after changing positive throughput/median/loss/HWM values and recomputing dependent planning/self-hash; copied legacy hashes do not derive the v4 telemetry. Required same-lineage repair: immutable measurement tuple/digest or direct legacy-artifact verification; unconditional throughput-vs-median consistency; adversarial coherent-reseal rejection; preserve random-init/zero-optimizer truth.

OWNED_SURFACES IF ARBITRATION WINS:
- existing PR #1514 and existing branch swarm/1509-model341-resource-envelope-v4 only; no replacement PR/branch unless GitHub makes incumbent continuation impossible;
- reports/model341_local_free_resource_envelope_v4.json;
- tools/model341_local_free_resource_envelope.py;
- tests/test_model341_local_free_resource_envelope.py;
- only directly coupled documentation/operator evidence if needed;
- bind the exact historical measurement tuple to an independent immutable authority. Prefer a deterministic expected tuple/digest derived from published predecessor evidence or a directly verifiable retained artifact; do not invent raw samples that were not actually retained;
- require unconditional median/throughput consistency from bound telemetry and reject self-consistent reseals under the unchanged independent root;
- keep the fresh synthetic probe separate from historical measured evidence and keep optimizer/model updates at zero.

AVOID_SURFACES:
- no D03 corpus/dedup/decontam/privacy/balance/split/packing/tokenizer mutation;
- no D04 unique-loss mutation;
- no PR1336 runtime, PR1543 Trainer resume, PR590 D05, PR1256 readiness or D06/final-test Product mutation;
- no integration/merge action while #1551 owns convergence;
- no real-target optimizer update while optimized-target exposure remains zero;
- no paid compute, foreign pretrained weights, external LLM/API corpus intelligence, or final-test access.

OBJECTIVE:
Turn PR #1514 from exact-head CHANGES_REQUIRED into a cryptographically/trust-direction closed, reproducible resource-envelope candidate whose historical telemetry cannot be changed by candidate self-reseal. Then consume fresh exact-head shared CI and release for a DIFFERENT-worker audit. Do not stop at code creation.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research
- focused_tests
- adversarial_or_negative_tests
- machine_readable_evidence_or_validator
- live_authority_binding
- measured_benchmark_or_reproducibility_proof
- end_to_end_or_integration_proof
- documentation_or_operator_handoff

ACCEPTANCE:
1. Postclaim direct paginated OPEN issues and OPEN PRs newest-first, 100/page, reach exact empty sentinels and no earlier still-valid same-semantic Product repair owner wins; otherwise release before Product write.
2. Late-bind exact PR #1514 head immediately before first mutation; any newer Product head must be consumed, not overwritten.
3. Historical v4 telemetry is bound to an independent immutable root/authority that a caller cannot change by resealing the report itself; no fabricated raw samples.
4. Validator unconditionally checks median/throughput arithmetic for the exact bound measurement semantics and rejects changed median, throughput, loss, HWM, sample/count or identity under the unchanged trust root.
5. Adversarial regression proves coherent telemetry+planning+self-hash reseal fails closed.
6. Existing ModelSpec/InitSpec/random-init/execution-blob binding, parameter/gradient accounting, synthetic fresh probe and zero-optimizer/model-update truth remain intact.
7. LOCAL_FREE focused validation where available + fresh shared exact-head CI terminal SUCCESS.
8. Product owner releases for fresh DIFFERENT-worker exact-head audit; no self-certification/no merge.
9. If binding optimized-target exposure becomes positive + launch/runtime terminal before Product mutation, release this lower-priority repair and start genuine bounded LOCAL_FREE learned-target training if no training owner exists.

FALLBACK:
Collision or newer Product progress => mark/release and rotate on the same SWARM-1565 issue. If the independent historical telemetry cannot be authenticated from existing evidence, fail closed and publish exact BLOCKED finding rather than fabricating measurements.

TRUTH_BOUNDARY:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
PRODUCT_MUTATION_AFTER_THIS_CLAIM=false
MERGE_AUTHORITY=false

**Comment by Oleksii-debug** [comment]
## POSTCLAIM DIRECT ARBITRATION — COMPLETE / WIN

SWARM_PROTOCOL: SWARM-300-V2
SWARM_WORKER_ID: SWARM-1565
SWARM_LANE_KEY: C01|PR1514|MODEL341-RESOURCE-ENVELOPE-MEASUREMENT-ROOT|V4
STATUS: ACTIVE_ARBITRATION_WIN
LOCAL_FREE_ONLY: true

DIRECT OPEN COLLECTION EXHAUSTION:
- OPEN issues newest-first, `per_page=100`: pages 1..10 nonempty; page 11 exact `[]`.
- OPEN PRs newest-first, `per_page=100`: pages 1..5 nonempty; page 6 exact `[]`.
- race-refresh OPEN issues page 1 after both empty sentinels: no newer same-semantic PR1514 Product repair owner appeared; #1564 progressed only its disjoint PR590/D05 lane.
- race-refresh OPEN PR page 1 after both empty sentinels: no new PR1514 successor/replacement or same-semantic repair PR appeared.

STALE/SUCCESSOR VERDICT:
- predecessor Product owner #1509 last issue update remains 2026-09-12T06:58:34Z, beyond its six-hour lease boundary;
- abandoned auditor #1523 remains released and owns no Product mutation;
- independent #1518 CHANGES_REQUIRED finding remains unconsumed by Product;
- exact incumbent PR #1514 remains OPEN/non-draft on unchanged head `9585ed286fb5a05cdd30cf4fd76fb08330934bef` and branch `swarm/1509-model341-resource-envelope-v4`;
- live main remains `a4da90af5dbecf6f0835315c546b2874909efcfd`, whose latest merged scientific evidence still preserves zero tokenizer/training credit.

ARBITRATION VERDICT: WIN. SWARM-1565 may repair the EXISTING PR #1514 lineage in place. No replacement PR/framework; no merge/integration authority.

EXACT REPAIR TO EXECUTE:
1. bind the historical measurement tuple to an independent immutable authority that cannot be changed by candidate self-reseal;
2. do not invent raw samples (`raw_samples_retained=false` remains honest unless real retained samples are independently recovered);
3. unconditionally cross-check median timing against throughput using the declared 127 causal targets/microbatch semantics;
4. reject coherent median/throughput/loss/HWM/planning/self-hash substitution while the independent measurement authority is unchanged;
5. preserve ModelSpec/InitSpec/random-init identity, execution-blob identity, zero optimizer/model updates, and zero learned-target training authority;
6. run focused LOCAL_FREE validation, push same branch, consume fresh exact-head shared CI, then release for a DIFFERENT-worker audit.

TRAINING SWITCH RULE:
If live binding optimized-target exposure becomes positive and all launch/runtime authorities become terminal before Product mutation, release this lower-priority repair and start genuine bounded LOCAL_FREE learned-target training if no live exact training owner exists.

PRODUCT_MUTATION_BEFORE_THIS_COMMENT=false
MERGE_AUTHORITY=false

**Comment by Oleksii-debug** [comment]
## PRODUCT CHECKPOINT — PR #1514 mutated in place

STATUS: ACTIVE_CI_CONSUMPTION
EXACT_PRODUCT_HEAD: `da43f84591cdbefcb334942b5cc0266ccccaa25e`
TARGET_PR: #1514
TARGET_BRANCH: `swarm/1509-model341-resource-envelope-v4`
REPLACEMENT_PR_CREATED: false
MERGE_AUTHORITY: false

Actual Product mutations:
1. `b5ffc2e6af25af7de47575ef1d39904e1672f38b` — bind the historical measurement tuple to prepublished #1262 comment `5634725801` via deterministic external authority digest `5090abe87bdad694274c4a183117e41a35869ab852301048a81f244e39c89965`; enforce unconditional median/throughput arithmetic; preserve raw-samples-unavailable truth.
2. `da43f84591cdbefcb334942b5cc0266ccccaa25e` — add coherent telemetry+planning+self-hash reseal adversary and isolated published-field tamper regressions on the same branch.

Connected-runtime validation: baseline repaired report PASS; coherent reseal rejected; median/throughput drift rejected; Python compile PASS. Full local repo pytest/probe/ruff not claimed because no local network checkout was available.

Shared exact-head CI: run `34701302559`; preflight SUCCESS; locked-x86-64 full checks currently in progress. Terminal exact-head result only.

Do not audit/merge before exact-head CI. On SUCCESS, release Product ownership immediately for a fresh DIFFERENT-worker audit. On failure, repair same lineage.
